plugins/github-pull-requests-board/src/hooks/useUserRepositories.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { useApi } from '@backstage/core-plugin-api';
+import { useEntity, catalogApiRef } from '@backstage/plugin-catalog-react';
+import { useCallback, useEffect, useState } from 'react';
+import { getProjectNameFromEntity } from '../utils/functions';
+
+export function useUserRepositories() {
+  const { entity: teamEntity } = useEntity();
+  const catalogApi = useApi(catalogApiRef);
+  const [repositories, setRepositories] = useState<string[]>([]);
+
+  const getRepositoriesNames = useCallback(async () => {
+    const entitiesList = await catalogApi.getEntities({
+      filter: {
+        kind: 'Component',
+        'spec.type': 'service',
```

Comment:
First impression - looks great @gregorytalita ! I was however wondering why you have hardcoded this plugin to only display components with type `service`? We use `website` and `library` a lot as well. In fact, I needed to remove this line to show anything at all for our own team in Backstage since we apparently only maintain other types :).

Why not just remove this line?

Reply:
@leon-vg https://github.com/backstage/backstage/issues/11888 😉

Reply:
Sounds perfect! Thanks for the response.